pkg/controller/framework_test.go

```diff
@@ -806,9 +807,10 @@ func newSnapshot(name, className, boundToContent, snapshotUID, claimName string,
 			SelfLink:        "/apis/snapshot.storage.k8s.io/v1alpha1/namespaces/" + testNamespace + "/volumesnapshots/" + name,
 		},
 		Spec: crdv1.VolumeSnapshotSpec{
-			Source: &crdv1.TypedLocalObjectReference{
-				Name: claimName,
-				Kind: "PersistentVolumeClaim",
+			Source: &v1.TypedLocalObjectReference{
+				Name:     claimName,
+				Kind:     "PersistentVolumeClaim",
+				APIGroup: &apiGrp,
```

Comment:
I think we make APIGroup optional. If it is an empty string, we don't need to specify it here. Could you try and confirm it?

Reply:
Yes.  We don't need to specify it.  I'll remove this.